README.md

```diff
@@ -207,6 +207,18 @@ if you need information about the registered id.
     }
   }
   ```
+* For remote references, if no version is specified, the latest version of schema is fetched and used from the schema registry.
+  You can also specify `-1` for the same functionality
```

Comment:
maybe we can move this part in the paragraph line 183 ?
```
If you have references to other schemas required before the register,
you can call the `addReference("name", "subject", version)`, this will add a reference to use from the registry.
The addReference calls can be chained.
``` ?

Reply:
Also, as this also work for compatibility, can you update the documentation for this part too ?